packages/components/src/Button/Component.purs

```diff
@@ -5,8 +5,6 @@ import Prelude
 import Data.Maybe (Maybe(..))
```

Comment:
Rename this file since it's not a Component anymore.

Reply:
What would be a good name for a purely functional component that is neither a classic nor hooks component?

Reply:
I think just `KSF.Button` is fine.

Reply:
Ah, now I get it! Done.